docs/_writing_macros.rst

```diff
@@ -248,32 +248,94 @@ The following expand declarations have replaced the original conditional element
           </options>
         </expand>
 
+From Galaxy release 22.01 named yields are supported. That is, if the macro contains
+``<yield name="xyz"/>`` it is replaced by the content of the ``token` tag with the same name.
+Token tags need to be direct children of the ``expand`` element.
+
```

Comment:
Can we have the example be something like:
Given the following macro:
...
and expanding the macro in the following way:
...
we get the following expanded definition:
...
?